stl/inc/__msvc_int128.hpp

```diff
@@ -73,7 +69,7 @@ struct
     }
 
     constexpr void _Right_shift(const unsigned char _Count) noexcept {
-        _STL_INTERNAL_CHECK(_Count < 128);
+        // _STL_INTERNAL_CHECK(_Count < 128);

```

Comment:
Should we remove them rather than commenting?

Reply:
They are preconditions. We could change the form of the comments, but we'd lose valuable info if we simply remove them. It's simple enough to search-and-replace these to enable/disable for maintenance, I'd prefer to just leave them here.

For posterity, the failure-reporting mechanisms for these assertions require linking to the libraries, so `_STL_INTERNAL_CHECK` isn't usable from core headers. (Notably this PR makes `_STL_INTERNAL_STATIC_ASSERT` usable in core headers.)